es2015.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     es6: true
   },
   extends: './es5.js',
+  parser: 'babel-eslint',
```

Comment:
Where were you seeing issues with this? I didn't think the es2015 config had any features that required a separate parser.